docs/operations/api-reference.md

```diff
@@ -519,9 +519,25 @@ Returns a list of objects of the currently active supervisors and their current
 |Field|Type|Description|
 |---|---|---|
 |`id`|String|supervisor unique identifier|
-|`state`|String|basic state of the supervisor. Available states:`UNHEALTHY_SUPERVISOR`, `UNHEALTHY_TASKS`, `PENDING`, `RUNNING`, `SUSPENDED`, `STOPPING`|
-|`detailedState`|String|supervisor specific state. (See documentation of specific supervisor for details)|
+|`state`|String|basic state of the supervisor. Available states: `UNHEALTHY_SUPERVISOR`, `UNHEALTHY_TASKS`, `PENDING`, `RUNNING`, `SUSPENDED`, `STOPPING`. Check [Kafka Docs](../development/extensions-core/kafka-ingestion.html#operations) for details.|
+|`detailedState`|String|supervisor specific state. (See documentation of the specific supervisor for details, e.g. [Kafka](../development/extensions-core/kafka-ingestion.html) or [Kinesis](../development/extensions-core/kinesis-ingestion.html))|
+|`healthy`|Boolean|true or false indicator of overall supervisor health|
+|`suspended`|Boolean|true or false indicator of whether the supervisor is in suspended state|
+
+* `/druid/indexer/v1/supervisor?fullStatus`
```

Comment:
Could the `fullStatus` API be unified with `/druid/indexer/v1/supervisor?full`?

It seems a bit confusing to have two "full" parameters, and the information returned by `fullStatus` appears to contain everything that `full` would return.

Reply:
`/druid/indexer/v1/supervisor?fullStatus` returns everything "full" returns, in addition adds:
- type
- source
- specString
these are added to fill the columns in `sys.supervisor` table without deserializing the `spec` object(which cannot be done in `SystemSchema` because of dependency issues). Also to avoid changing the behavior of existing API.

Reply:
Can you add an explanation to the docs describing why the `fullStatus` param exists and when you would use one vs. the other

Reply:
Since `fullStatus` is used internally for sys tables, i'm now thinking should we document it, as I think for users, the `full` query param might suffice. Not sure what is generally our stance on these kind of internal options. I think, the only reason, one would use `fullStatus` , is if they care to get `type` or `source` without digging into the `spec` object. I can either remove the `fullStatus` or add that explanation. Thoughts ?

Reply:
IMO, it's ok to skip documentation for options meant for internal use. Maybe call it `?system` rather than `?full` to make it clearer in the code what the purpose is.

Reply:
`?system` sounds good to me and remove the option from docs.

Reply:
I think it's also better to skip the documentation. If it's intended for internal use and folks start using it because it's documented, it may be harder to change the behavior/API later if needed.

Reply:
Got it, if it's only meant for internal use, then renaming `fullStatus` to `system` and leaving out the docs for that sounds good to me.